design-editor/src/templates/circle-helper.js

```diff
@@ -27,7 +27,8 @@ function onShow(event) {
 		toggleSwitches: new Component("input[data-appearance]", tau.widget.ToggleSwitch),
 		dimmers: new Component(".ui-dimmer", tau.widget.Dimmer),
 		graphs: new Component(".ui-graph", tau.widget.Graph),
-		progressBars: new Component(".ui-circle-progress", tau.widget.CircleProgressBar)
+		progressBars: new Component(".ui-circle-progress", tau.widget.CircleProgressBar),
+		selector: new Component(".ui-section-changer", tau.widget.SectionChanger)
```

Comment:
Please change key name to "sectionChanger" or something similar

Reply:
Thanks. Changed to "sectionChanger".